server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -1,142 +1,69 @@
-  --pull ${PB_ELASTICSEARCH_IMAGE_PULL_POLICY} \
-  --umask 0000 \
-  --restart no \
-  --pids-limit 0 \
-  -e PATH=/usr/share/elasticsearch/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
-  -e HOME=/root \
-  -e TERM=xterm \
-  -e container=oci \
-  -e discovery.type=single-node \
-  -e ES_JAVA_OPTS="-Xms1g -Xmx1g"\
-  -e xpack.security.enabled=false \
-  -e ES_PORT=9200 \
-  -d \
-  ${PB_ELASTICSEARCH_IMAGE} \
-  /bin/tini -- /usr/local/bin/docker-entrypoint.sh eswrapper
-
-podman run \
-  --rm \
-  --pod ${PB_POD_NAME} \
-  --name "${PB_POD_NAME}-postgres" \
-  --workdir /opt/app-root/src \
-  --pull ${PB_POSTGRESQL_IMAGE_PULL_POLICY} \
-  --umask 0000 \
-  --user 26:26 \
-  --restart no \
-  --pids-limit 0 \
-  --entrypoint container-entrypoint \
-  -e POSTGRESQL_USER=pbenchcontainer \
-  -e POSTGRESQL_PASSWORD=pbench \
-  -e POSTGRESQL_DATABASE=pbenchcontainer \
-  -e HOME=/var/lib/pgsql \
-  -e CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/postgresql \
-  -e APP_ROOT=/opt/app-root \
-  -e POSTGRESQL_VERSION="13" \
-  -e PGUSER=postgres \
-  -e STI_SCRIPTS_URL=image:///usr/libexec/s2i \
-  -d \
-  ${PB_POSTGRESQL_IMAGE} \
-  run-postgresql -p 5432
-
-podman run \
-  --rm \
-  --pod ${PB_POD_NAME} \
-  --name "${PB_POD_NAME}-keycloak" \
-  --pull ${PB_KEYCLOAK_IMAGE_PULL_POLICY} \
-  --umask 0000 \
-  --restart no \
-  --pids-limit 0 \
-  -e KEYCLOAK_ADMIN=admin \
-  -e KEYCLOAK_ADMIN_PASSWORD=admin \
-  -d \
-  ${PB_KEYCLOAK_IMAGE} \
-  start-dev --health-enabled=true --http-port=8090
-
-server/pbenchinacan/load_keycloak.sh
-
-# We expect the host dashboard directory already exists, however,
-# in case it is missing we will create an new empty directory.
-# Note: if the dashboard directory is not present Pbench Server runs
-# without a dashboard deployment.
-mkdir -m 0755 -p ${PB_DASHBOARD_DIR}
-
-podman run \
-  --rm \
-  --hostname pbenchinacan \
-  --network host \
-  --name "${PB_POD_NAME}-pbenchserver" \
-  --pull ${PB_SERVER_IMAGE_PULL_POLICY} \
-  --umask 0000 \
-  --restart no \
-  --pids-limit 0 \
-  --entrypoint /sbin/init \
-  --workdir / \
-  -e PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
-  -v ${PB_DASHBOARD_DIR}:/srv/pbench/public_html/dashboard:z \
-  -d \
-  ${PB_SERVER_IMAGE}
+# Set up TMP_DIR, if it's not already defined, to point to WORKSPACE_TMP, if it
+# is defined (e.g., by the CI), or to `/var/tmp/pbench` as a fallback.
+#
+# Point SRV_PBENCH at a file tree in the temporary space which will be mapped
+# into the Pbench Server container as /srv/pbench to provide all the storage for
+# the Server and the Dashboard.
+#
+# Point PB_DEPLOY_FILES at a directory in the temporary space to hold the
+# Pbench Server deployment files (e.g., config files).
+TMP_DIR=${TMP_DIR:-${WORKSPACE_TMP:-/var/tmp/pbench}}
+export SRV_PBENCH=${SRV_PBENCH:-${TMP_DIR}/srv/pbench}
+export PB_DEPLOY_FILES=${PB_DEPLOY_FILES:-${TMP_DIR}/pbench_server_deployment}
```

Comment:
We're not clearing these directories out before using them...should we be?  (Do we care?)

Reply:
We should for the CI.

Reply:
Alright, I'll put up another PR for that (since this PR seems to have reached consensus).